app/workers/cassandra_writer.rb

```diff
@@ -0,0 +1,15 @@
+class CassandraWriter
+  include Sneakers::Worker
+  from_queue "worker.cassandra"
+
+  def work(event_json)
+    attributes = JSON.parse(event_json)
+
+    Query.create!(
+      agent_id: attributes['agent_id'],
+      path: attributes['data']['path'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.